tests/embedding/test_app.py

```diff
@@ -52,3 +54,67 @@ def test_app_version(embedded_app):
     version = embedded_app.version
     assert type(version) is int
     assert version >= 231
+
+
+@pytest.mark.embedding
+def test_warning_message():
+    """Test pythonnet warning of the embedded instance."""
+    venv_name = "pythonnetvenv"
+    base = os.getcwd()
+
+    if "win" in sys.platform:
+        exe_dir = "Scripts"
+    else:
+        exe_dir = "bin"
+
+    venv_bin = os.path.join(base, "." + venv_name, exe_dir)
+
+    # Set up path to use the virtual environment
+    original_path = os.environ["PATH"]
+    os.environ["PATH"] = venv_bin + os.pathsep + os.environ.get("PATH", "")
+
+    # Create virtual environment
+    subprocess.run([sys.executable, "-m", "venv", "." + venv_name])
+
+    # Upgrade pip
+    print("Upgrading pip")
```

Comment:
I'd personally  remove the ``print`` statements. They are fine for local dev - but do not make them part of the PR.

Reply:
Sure, that makes sense! I'll remove them